main.go

```diff
@@ -56,6 +59,10 @@ func runMain() error {
 	log.SetFlags(0) // Removes default timestamp flag
 
 	args := procArgs()
+	if args.Version {
+		log.Println(version)
+		return nil
+	}
```

Comment:
I think we should always log the version but if -v is passed it should then immediately exit.